profile.md

```diff
@@ -232,6 +233,12 @@ part of a Kubernetes API call:
   }
   ```
 
+## Platform to Service Broker Authentication
+
+The [specification](spec.md) requires that brokers implement HTTP Basic authentication, but also allows for other mechanisms if the platform and the broker agree upon them outside of the specification. This section contains a list of supported mechanisms that platforms and brokers support. The list will be growing as time goes on.
+
+- The [Kubernetes Service Catalog](https://github.com/kubernetes-incubator/service-catalog) supports arbitrary HTTP Bearer tokens over TLS as an authentication mechanism
```

Comment:
I'm not sure we want to add this to the spec - it feels kind of awkward since each time an impl changes what it  supports we'd need to publish a new version of the spec. Putting it in our wiki might be ok though.

Reply:
If we wanted to add something to the Profile about auth, then I think having auth-specific rules or guidance would be appropriate - stuff that all impls should adhere too.

Reply:
As a service author, where can I go to learn how to build a broker against this? The service-catalog readme doesn't direct me anywhere to learn this

Reply:
This is why the info here shouldn't really be platform specific, rather, if anything, it would be auth specific. E.g. I don't think svc-cat's use of HTTP bearer tokens is unique, or at least I hope not, otherwise we wouldn't have interop. 

I kind of wonder what the purpose of this text is. I don't think we should just reiterate what each platform's docs say w.r.t. which auth mechanisms it supports. That's not only duplicate info but requires us to keep things in sync. I'd prefer for this section to be more about any OSB API specific aspects of the various auth mechanisms Platforms support that people will need to know about if they want to write code - that way the odds of having interop go up.

Reply:
re [Matt's comment](https://github.com/openservicebrokerapi/servicebroker/pull/398#discussion_r164432525)
>As a service author, where can I go to learn how to build a broker against this? The service-catalog readme doesn't direct me anywhere to learn this

We've got a [general documentation issue](https://github.com/kubernetes-incubator/service-catalog/issues/1678) we hope to address for the next major release which will help, but you are right, presently there isn't much doc on this

Reply:
+1 to ^^ - we don't have much to put here yet, but it would be nice to set the precedent. if we don't want to put specific systems in here, I would be ok with something like a "coming soon" in this section

Reply:
Per offline conversation with @duglin, @jboyd01 and myself, we will:

- move this section into the spec.md, replacing it with the link to profile.md
- link to a wiki page which @duglin will create
- the wiki page will be non-normative and contain at least a non-exhaustive list of known platforms (and possibly brokers in the future?) that implement other auth. mechanisms